### ISSUE TITLE
Make visualization in utils

### ISSUE THREAD

**Issue by MijeongJeon** [KerasKorea/KerasObjectDetector#4]
Visualization Code in Utils

Including
- Draw bounding box on image
- Visualize boxes labels on image

**Comment by MijeongJeon** [comment]
[Reference] 
https://github.com/tensorflow/models/blob/master/research/object_detection/utils/visualization_utils.py
